app/src/main/java/com/appunite/loudius/ui/components/LoudiusFullScreenError.kt

```diff
@@ -155,3 +104,51 @@ fun LoudiusErrorScreenPreview() {
         )
     }
 }
+
+@Preview(showSystemUi = true, device = ReferenceDevices.SmallPhone)
+@Composable
+fun LoudiusErrorScreenPreviewSmallPhone() {
+    LoudiusTheme {
+        LoudiusFullScreenError(
+            errorText = stringResource(id = R.string.error_dialog_text),
+            buttonText = stringResource(R.string.try_again),
+            onButtonClick = {},
+        )
+    }
+}
+
+@Preview(showSystemUi = true, device = ReferenceDevices.SmallPhoneLandscape)
+@Composable
+fun LoudiusErrorScreenPreviewSmallPhoneLandscape() {
```

Comment:
AFAIK they recently added multi-preview annotation. I think it could work here well.
[LINK](https://developer.android.com/jetpack/compose/tooling/previews#preview-multipreview)